sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessorClient.cs

```diff
@@ -352,10 +352,10 @@ public class EventProcessorClient : IAsyncDisposable
         ///   external mocking.
         /// </remarks>
```

Comment:
Maybe we should update this remark given that the constructor is `protected` now.